src/main/java/io/zeebe/monitor/zeebe/ZeebeHazelcastService.java

```diff
@@ -38,6 +40,10 @@ public void start() {
     connectionRetryConfig.setClusterConnectTimeoutMillis(
         Duration.parse(hazelcastConnectionTimeout).toMillis());
 
+    if(hazelcastClusterName != null) {
```

Comment:
that check for != null is not required, since default application.yaml contains a proper setting. for consistency as with other settings, I recommend dropping that.